lib/views.js

```diff
@@ -438,7 +437,7 @@ exports.tasks_messages = {
             // used for fetching a specific message based on uuid
             emit(msg.uuid, val);
             // used for querying latest tasks in a specific state
-            emit([task.state, when], val);
+            emit(task.state, val);
```

Comment:
NB - the [API documentation](https://github.com/medic/medic-api#get-apiv1messages) says:

> Returns list of messages, oldest first based on timestamp or due date.

I don't think this is true any more.